react/OrderItems.tsx

```diff
@@ -177,12 +177,12 @@ const useEnqueueTask = () => {
         },
         (error: any) => {
           if (error && error.code === constants.TASK_CANCELLED_CODE) {
-            popLocalOrderQueue(error.index)
+            popLocalOrderQueue(error.index, { log: logSplunk })
```

Comment:
why not pass `logSplunk` directly?

Reply:
you mean like
```suggestion
            popLocalOrderQueue(error.index, { logSplunk })
```

or
```suggestion
			popLocalOrderQueue(error.index, logSplunk)
```
?

Reply:
the second one

Reply:
hmmm
my line of thought was that we might need to pass more params in the future, but now that you ask, I realize that this shouldn't matter, as this isn't a public api

Reply:
so sure, I can do that

Reply:
exactly hahah 😄 we can change this anytime if we feel the API is becoming a bit awkward

Reply:
ok, will do like that when implementing on the pkg ✨